SPECS/cmake/cmake.spec

```diff
@@ -21,6 +21,7 @@ Patch6:         CVE-2023-38545.patch
 Patch7:         CVE-2023-38546.patch
 Patch8:         cve-2023-44487.patch
 Patch9:         CVE-2023-28320.patch
+Patch10:        CVE-2023-27533.patch
```

Comment:
What about CVE-2023-27534?

Reply:
Thanks @jslobodzian. I have taken this PR changes into #10484 and hence closing this PR.